app/src/main/java/com/example/kuba10/mypokemonplaces/ListFragmentAdapters/AdapterForTouchAndFirebase.java

```diff
@@ -1,186 +0,0 @@
-package com.example.kuba10.mypokemonplaces.ListFragmentAdapters;
-
-import android.view.MotionEvent;
-import android.view.View;
-import android.widget.ImageView;
-
-import com.example.kuba10.mypokemonplaces.Constants;
-import com.example.kuba10.mypokemonplaces.ListFragment.FirebaseListFragment;
-import com.example.kuba10.mypokemonplaces.Model.PokePlace;
-import com.example.kuba10.mypokemonplaces.Model.PokemonGo;
-import com.example.kuba10.mypokemonplaces.PlaceDetailsFragment.PlaceDetailsFragment;
-import com.example.kuba10.mypokemonplaces.R;
-import com.firebase.ui.database.FirebaseRecyclerAdapter;
-import com.google.firebase.database.ChildEventListener;
-import com.google.firebase.database.DataSnapshot;
-import com.google.firebase.database.DatabaseError;
-import com.google.firebase.database.DatabaseReference;
-import com.google.firebase.database.Query;
-import com.squareup.picasso.Picasso;
-
-import java.util.ArrayList;
-import java.util.Collections;
-
-public class AdapterForTouchAndFirebase extends FirebaseRecyclerAdapter<PokePlace, CardViewHolder> implements ItemTouchHelperAdapter {
-
-    private final Query ref;
-    private ChildEventListener mChildEventListener;
-    private ArrayList<PokePlace> userPlacesDataList;
-    private ArrayList<PokemonGo> pokemonGo_data_list;
-    private FirebaseListFragment parentFragment;
-    private OnStartDragListener mOnStartDragListener;
-
-    public AdapterForTouchAndFirebase(Class<PokePlace> modelClass, int modelLayout,
-                                      Class<CardViewHolder> viewHolderClass,
-                                      Query ref, OnStartDragListener onStartDragListener, FirebaseListFragment fragment) {
-        super(modelClass, modelLayout, viewHolderClass, ref);
-        this.ref = ref;
-
-        mOnStartDragListener = onStartDragListener;
-        userPlacesDataList = new ArrayList<>();
-        parentFragment = fragment;
```

Comment:
Postaraj się nazywać pola i parametry konstruktora tak samo, potem korzystać ze słowa kluczowego `this` - tak jest bardziej "po javovemu"

Reply:
ok